frontend/src/App.js

```diff
@@ -43,15 +43,15 @@ function App() {
 
 	let mainContainerWidth = `${Math.min(width, 1168)}px`;
 
-	useEffect(() => {
-		appwriteApi.getAccount()
-			.then((r) => {
-				setIsLoaded(true);
-				setUser(r);
-			})
-			.catch(() => {
-				setIsLoaded(true);
-			});
+	useEffect(async () => {
```

Comment:
Maybe passing an async lambda is not such a good idea. In the RestrictedArea Component, doing that gave a React state update warning.